test/fakeintake/server/body.go

```diff
@@ -0,0 +1,29 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package server
+
+type errorResponseBody struct {
+	Errors []string `json:"errors"`
+}
+
+// Same struct as in datadog-agent/comp/core/flare/helpers/send_flare.go
```

Comment:
No strong reason here. Just did not want to export the struct in `comp/core/flare/...` since I assumed that the fake intake should not be coupled with the main agent (even if it probably already is) + I don't think it's likely that the backend response format will change so duplication should not be really painful imho.

If you think it's better to use the struct in `datadog-agent/comp/core/flare/helpers/send_flare.go` I can change it

Reply:
At least you should add a comment in `comp/core/flare/` saying that any changes to the struct must also be applied here.

Reply:
Done in 97ad04fae7531af880b31be1fd6cc7317eac9976